controller/src/main/java/io/pravega/controller/server/AuthResourceRepresentation.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server;
+
+import io.pravega.common.Exceptions;
+
+/**
+ * A utility class with methods for preparing string representations of auth-protected resources.
+ * <p>
+ * Background:
+ * <p>
+ * In general, authorization is about granting a <i>subject</i> access to perform a particular <i>action</i>
+ * on an <b>object/resource</b>.
+ * <p>
+ * In Pravega,
+ * <ul>
+ *     <li>A subject is represented as an instance of type {@link java.security.Principal}.</li>
+ *     <li>An action is represented as an element of enum type {@link io.pravega.auth.AuthHandler.Permissions}.</li>
+ *     <li>An object is represented by an instance of <i>this</i> class.</li>
+ * </ul>
+ */
+public final class AuthResourceRepresentation {
+
+    /**
+     * Creates a resource representation for use in authorization of actions pertaining to the collection of scopes
+     * in the system.
+     *
+     * @return a string representing the collections of scopes in the system
+     */
+    public static String ofScopes() {
+        return "/";
+    }
+
+    /**
+     * Creates a resource representation for use in authorization of actions pertaining to the specified scope.
+     *
+     * @param scopeName the name of the scope
+     * @return a string representing the scope with the specified name
+     * @throws NullPointerException if {@code scopeName} is null
+     * @throws IllegalArgumentException if {@code scopeName} is empty
+     */
+    public static String ofAScope(String scopeName) {
+        Exceptions.checkNotNullOrEmpty(scopeName, "scopeName");
+        return scopeName;
+    }
+
+    /**
+     * Creates a resource representation for use in authorization of actions pertaining to the collection of streams
+     * within the specified scope.
+     *
+     * @param scopeName the name of the scope
+     * @return a string representing the collection of streams within the scope with the specified name
+     * @throws NullPointerException if {@code scopeName} is null
+     * @throws IllegalArgumentException if {@code scopeName} is empty
+     */
+    public static String ofStreamsInScope(String scopeName) {
+        Exceptions.checkNotNullOrEmpty(scopeName, "scopeName");
```

Comment:
FYI, you can just return the result of this method. It returns the argument you passed.

Reply:
Ah! I didn't realize that. Will fix it.